notebooks/GPU.ipynb

```diff
@@ -36,7 +36,7 @@
       "outputs": [],
       "source": [
         "#@title Select your model below, then click the play button to start the UI.\n",
-        "#@markdown Afterwards, just sit tight and wait - the link to the UI should show up after it's done starting up.\n",
+        "#@markdown Afterwards, just sit tight and wait - the link to the UI should show up after it's done starting up.\nWARNING: The 6B Experimental model can currently produce broken outputs due to a bug in Kobold. We recommend using the standard Pygmalion 6B model until it is fixed.\n",
```

Comment:
I'd be more stern with this wording and straight up say it currently produces broken outputs, not that it _can_.

Reply:
Done.